homeassistant/components/just_nimbus/sensor.py

```diff
@@ -0,0 +1,180 @@
+"""Support for the CO2signal platform."""
+from __future__ import annotations
+
+from dataclasses import dataclass
+from datetime import timedelta
+
+import justnimbus
+
+from homeassistant.components.sensor import SensorDeviceClass, SensorEntityDescription
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.const import (
+    CONF_CLIENT_ID,
+    PRESSURE_BAR,
+    TEMP_CELSIUS,
+    VOLUME_LITERS,
+)
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers.entity_platform import AddEntitiesCallback
+from homeassistant.helpers.typing import StateType
+
+from . import JustNimbusCoordinator
+from .const import DOMAIN, VOLUME_FLOW_RATE_LITERS_PER_MINUTE
+from .entity import JustNimbusEntity
+
+SCAN_INTERVAL = timedelta(minutes=3)
+
+
+@dataclass
+class JustNimbusEntityDescription(SensorEntityDescription):
+    """Provide a description of a JustNimbus sensor."""
+
+    unique_id: str | None = None
+
+
+SENSOR_TYPES = (
+    JustNimbusEntityDescription(
+        key="pump_flow",
+        name="Pump Flow",
+        icon="mdi:pump",
+        native_unit_of_measurement=VOLUME_FLOW_RATE_LITERS_PER_MINUTE,
+        unique_id="jn:pump_flow",
+    ),
+    JustNimbusEntityDescription(
+        key="drink_flow",
+        name="Drink Flow",
+        icon="mdi:water-pump",
+        native_unit_of_measurement=VOLUME_FLOW_RATE_LITERS_PER_MINUTE,
+        unique_id="jn:drink_flow",
+    ),
+    JustNimbusEntityDescription(
+        key="pump_pressure",
+        name="Pump Pressure",
+        icon="mdi:arrow-collapse-vertical",
```

Comment:
The frontend provides a default icon for the pressure device class, there is no need to override that.

```suggestion
```